tests/v1/kv_connector/nixl_integration/run_accuracy_test.sh

```diff
@@ -130,7 +136,7 @@ run_tests_for_model() {
     vllm serve $model_name \
     --port $PORT \
     --enforce-eager \
```

Comment:
`--enable-log-requests` could be useful here if you're doing another update

Reply:
yep still waiting for CI to report

Reply:
allow me to postpone this change, I think I should probably silence the progress bar from lm-eval first because it's already way too verbose given it will show outputs from up to 4 instances all at once..